examples/Esp8266-mqtt/ciotc_config.h

```diff
@@ -0,0 +1,72 @@
+/******************************************************************************
+ * Copyright 2018 Google
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+// This file contains your configuration used to connect to Cloud IoT Core
+
+// Wifi newtork details.
+const char *ssid = "SSID";
+const char *password = "PASSWORD";
+
+// Cloud iot details.
+const char *project_id = "project-id";
+const char *location = "us-central1";
+const char *registry_id = "my-registry";
+const char *device_id = "my-esp32-device";

```

Comment:
rename to `my-esp8266-device` to clarify this is not for the ESP32

Reply:
done.